tests/test_mpi.py

```diff
@@ -1094,7 +1093,7 @@ class TestIsotropicAcoustic(object):
     """
 
     @pytest.mark.parametrize('shape,kernel,space_order,nbpml,save,Eu,Erec,Ev,Esrca', [
-        ((60, ), 'OT2', 4, 10, False, 976.825, 9372.604, 18851836.075, 47002871.882),
+        ((60, ), 'OT2', 4, 10, False, 385.853, 12937.250, 63818503.321, 101159204.362),
```

Comment:
why are these changing, but not the ones below?

Reply:
Because the previous setup (don't know when changed) was messing up 1D only because the if dim>1 was removed. Once back 1D needs new values